.github/workflows/build.yml

```diff
@@ -8,8 +8,9 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
-    - uses: ljharb/actions/node/install@main
-      name: 'nvm install lts/* && npm install'
+    - uses: actions/setup-node@v4
```

Comment:
this is not a preferred change, it's actually less simple, and it's not maintained by me so tc39 has less control over it.

Reply:
It's the simplest because everyone uses it, because it manages versions for us, and it hides the npm dep.

Reply:
why would we want npm hidden?

Reply:
my bad I mean `cache`.
But FIY "cache" in French mean hide. So just ADHD

Reply:
my actions can do caching as well, but caching adds complexity that can hide bugs, so it's usually best avoided when speed isn't a priority, as it isn't here.

Reply:
just realize that caching can be accomplish without lock file

Reply:
Let's get back to the main topic, which is that `action/setup-node` is used by everyone and is very well maintained.
And I remain convinced that using a standard GHA is the right approach.

Reply:
It is not always well maintained - there's been problems with it before - and it doesn't really matter who else uses it. We're going to keep using the same things we've been using across the tc39 org, they work well and we can directly fix issues in them, unlike in setup-node.